app/resources/admin.html.template

```diff
@@ -138,35 +178,80 @@
     var title_col = row.insertCell(cur_col++);
     var title = document.createElement('input');
     title.type = 'text';
-    title.id = 'language_title_' + iteration;
+    title.id = 'repo_titles_language_value_' + iteration;
     title_col.appendChild(title);
-
     return iteration;
   }
 
+  function add_language_span(table_id, language) {
```

Comment:
The name of this function is a bit confusing because it doesn't just add a span. Can you rename this to e.g., add_language_row_with_span?